src/app/components/SubjectHeading/Search/SubjectHeadingSearch.jsx

```diff
@@ -38,15 +38,16 @@ class SubjectHeadingSearch extends React.Component {
 
   makeApiCallWithThrottle(timerId) {
     const apiCall = () => {
-      if (this.state.userInput.length) return axios(`${appConfig.baseUrl}/api/subjectHeadings/autosuggest?query=${this.state.userInput}`)
+      if (this.state.userInput) return axios(`${appConfig.baseUrl}/api/subjectHeadings/autosuggest?query=${this.state.userInput}`)
```

Comment:
Makes sense, are there any particular cases this is meant to catch?

Reply:
The conditional logic? It check for truthiness `userInput`, because if it's empty or undefined, we don't want to make an API call and no results should be shown.